docs/API.md

```diff
@@ -451,43 +466,29 @@ Example response:
 
 An object with the following properties:
 
-* `"client_id"`: A free-form string, optional.
-* `"domain"`: A *domain name*, required.
-* `"profile"`: A *profile name*, optional.
-* `"client_version"`: A free-form string, optional.
-* `"nameservers"`: A list of *name server* objects, optional.
-* `"ds_info"`: A list of *DS info* objects, optional.
+* `"client_id"`: A free-form string, optional. Used to monitor which client uses the API.
+* `"domain"`: A *domain name*, required. The zone to test.
```

Comment:
We should have restrictions on what code points that the domain may consist of. And a lenght limit.

Reply:
@matsduf Check the *domain name* data type specification.